aerogear/mobile-security-service-operator
# feat(Unbind) : Impl of unbind CR and operations

## Motivation
* https://issues.jboss.org/browse/AEROGEAR-8916

## What
- Impl to unbind the apps
- Update the readme

## Verification Steps
1. Install the operator
2. Create a namespace and bind an app by the command (make bind)
3. In the same namespace unbid the app by the command (make unbind) and check that the app was deleted/disable in the Service

After do and undo the bind and unbind many times to check if all will work well as expected.

<img width="839" alt="Screenshot 2019-04-13 at 15 28 46" src="https://user-images.githubusercontent.com/7708031/56081467-89b99880-5e05-11e9-86c6-a646d79f1f38.png">

<img width="917" alt="Screenshot 2019-04-13 at 15 29 00" src="https://user-images.githubusercontent.com/7708031/56081466-89b99880-5e05-11e9-939c-86ca00625275.png">

## Checklist:


## Progress


## Additional Notes
